### ISSUE TITLE
Add config.yaml validation

### ISSUE THREAD

**Issue by soniamitchell** [FAIRDataPipeline/FAIR-CLI#114]
For the working config file:
```
run_metadata:
  default_input_namespace: smitchell
  default_output_namehspace: test
  default_output_namespace: smitchell
  description: Simple Model
  latest_commit: f056d94a12860cd9b4a7654732d3bb87803a4ae9
  local_data_registry_url: http://localhost:8000/api/
  local_repo: /Users/Soniam/Desktop/git/FAIRDataPipeline/rDataPipeline
  public: true
  remote_data_registry_url: https://data.scrc.uk/api/
  remote_repo: https://github.com/FAIRDataPipeline/rDataPipeline.git
  script: saldkhj
  write_data_store: /Users/Soniam/.fair/data
```
the command line interface should initially perform key checking as a first stage of validation.

Also consider future implementation of a validator via a schema using the Cerberus or Pydantic python modules.

Editted by @kzscisoft 15/11/2021

**Comment by kzscisoft** [comment]
This is actually a pretty big feature and falls more under the heading of creating a complete validator for the config.yaml file, you can use either `pydantic` or `cerberus` python modules to define a schema which must be followed.

**Comment by kzscisoft** [comment]
Fixes #122
